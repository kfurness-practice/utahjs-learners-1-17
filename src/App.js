import React, { Component } from 'react';
// import logo from './logo.svg';
import Link from './Link.react';
// import  './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <div className="App-header">

          <h2>To-Do Demo App</h2>
        </div>
        <Body />
        <Link />
      </div>
    );
  }
}

export default App;

export const Body = () => {
  return (
    <div>
      <h2>This is the Body!!!!!!!!</h2>
      <h3>More stuff will go here</h3>
    </div>
  )
}
