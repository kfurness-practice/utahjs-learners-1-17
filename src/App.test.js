import React from 'react'
import App, { Body } from './App'
import renderer from 'react-test-renderer'

test('Search snapshot test', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})

test('Body snapshot test', () => {
  const component = renderer.create(<Body />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
