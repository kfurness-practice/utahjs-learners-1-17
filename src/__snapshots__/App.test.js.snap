exports[`test Body snapshot test 1`] = `
<div>
  <h2>
    This is the Body!!!!!!!!
  </h2>
  <h3>
    More stuff will go here
  </h3>
</div>
`;

exports[`test Search snapshot test 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <h2>
      To-Do Demo App
    </h2>
  </div>
  <div>
    <h2>
      This is the Body!!!!!!!!
    </h2>
    <h3>
      More stuff will go here
    </h3>
  </div>
  <a
    className="normal"
    href="#"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]} />
</div>
`;
